stories/menu.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { checkA11y } from 'storybook-addon-a11y';
+import { withInfo } from '@storybook/addon-info';
+import { Menu, MenuItem, MenuDivider } from '../components';
+
+storiesOf('Menus', module)
+  .addDecorator((story, context) => withInfo('common info')(story)(context))
+  .addDecorator(checkA11y)
+  .add('Menu', () => (
+    <Menu onSelect={action('select')} selectable={false} selected>
+      <MenuItem value="foo" caption="Caption" />
+      <MenuItem onClick={action('click')} value="bar" caption="Caption & Shortcut" shortcut="Ctrl + P" />
+      <MenuItem caption="Disabled ..." disabled shortcut="Ctrl + P" />
+      <MenuDivider />
+      <MenuItem caption="Caption & Icon" icon="check" />
+      <MenuItem caption="Caption, Icon & Shortcut" icon="list_bulleted" shortcut="Ctrl + P" />
+      <MenuItem caption="Disabled ..." icon="flag" shortcut="Ctrl + P" disabled />
+    </Menu>
+  ));
```

Comment:
Not sure what's happening here, but it's not rendering good at all.

Reply:
The problem with this icon menu items are that they rely on an icon font for the icons. This icon font is not included in the component styles, but needs to be loaded separate.

This icon font should be replaced with svg icons in the near future, so I would rather not include them in the storybook at the moment :)

Reply:
Ok. The menu will indeed have to change to using our icon components instead of the font.

Reply:
resolved?